java/AccelerometerParser.java

```diff
@@ -114,9 +116,6 @@ public static void main(String[] args) {
 					verbose = Boolean.parseBoolean(funcParam.toLowerCase());
 				} else if (funcName.equals("epochPeriod")) {
 					epochPeriod = Integer.parseInt(funcParam);
-				// Make sure time f
-				} else if (funcName.equals("timeFormat")) {
-					timeFormat = DateTimeFormatter.ofPattern(funcParam);
```

Comment:
Remove the ability to specify a time format as this will mess up with Panda's `date_parser`